operations/mimir-mixin-compiled-baremetal/alerts.yaml

```diff
@@ -96,8 +96,8 @@ groups:
       runbook_url: https://grafana.com/docs/mimir/latest/operators-guide/mimir-runbooks/#mimirmemcachedrequesterrors
```

Comment:
I think you should rename the alertname, runbook and message too because right now they're memcached specific